console.log("Merge conflict resolved")

func = () => {
    return 5+10
}

console.log(func())
